src/app/pages/send/send-crypto-asset-form/_components/stacks-recipient-field.tsx

```diff
@@ -0,0 +1,16 @@
+import { useStacksRecipientBnsName } from '../_hooks/use-stacks-recipient-bns-name';
+import { RecipientField } from './recipient-field';
+import { RecipientFieldBnsAddress } from './recipient-field-bns-address';
+
+export function StacksRecipientField() {
```

Comment:
This component is stacks specific, so could better live under `src/app/pages/send/send-crypto-asset-form/stx/components`.

Reply:
This isn't just used by `stx`, it is also used by `stacks` ft's, so that is why I proposed earlier to do the routing/folder structure as `/blockchain/symbol`. We can def change to that if you are ok with it?

Reply:
Got it. I'm for this suggestion, I know Ledger Live work with blockchain "families" 👨‍👧‍👦

Reply:
```
send/
├── components/
│   ├── family/
│   │   ├── stacks
│   │   └── bitcoin
│   └── generic/
│       └── other components
└── form/
    ├── btc
    ├── stx
    └── sip10
``` 

?

Reply:
I personally think it makes more sense to structure the `family` you are showing at a higher level. Here, you would also have to create a `family` and `generic` folder for shared hooks. Also, the structure you are showing isn't totally accurate in that under send their is a page to select the asset and a form folder which contains the `components` and `hooks` for forms.

Reply:
so, v2:

```
send
├── choose-crypto-asset
└── send-crypto-asset-form
    ├── hooks
    ├── components
    ├── family
    │   ├── stacks
    │   │   ├── hooks
    │   │   └── components
    │   └── bitcoin
    │       ├── hooks
    │       └── components
    └── form
        ├── btc
        ├── stx
        └── sip10
```
?

Reply:
Looks better to me. I don't totally understand using 'family' instead of 'blockchain' but fine to run with it. We had `component families` in modeling architecture so gives me the creeps. 🤣

Reply:
I'm not super against blockchain, but guess Ledger went for family as multiple blockchains can be in the same "family". Perhaps there are more turns of phrase we can borrow from modelling architecture lol

Reply:
Made the changes, but wondering if `family` and `form` should actually be plural?